### PR TITLE
Add response-driven reminder pack code mappings

### PR DESCRIPTION
**Pull request by liamtoozer**
# Motivation and Context

- [x] Updated ops tool with any new scheduled action types?
# Motivation and Context

RM will be given lists of LSOAs to generate response-driven reminders for. This PR adds new action types / pack codes.

# What has changed

* Added new pack codes
* Explicitly ignore (unused) cryptography package warning for now. We will need to update the package at some point so will raise a ticket for this.

# How to test?

See Toolbox PR

# Links

https://trello.com/c/tRiFtYj5/984-reminders-response-driven-8

### CODE REVIEW

**Comment by liamtoozer** [comment]
[Reminders - response driven (8)](https://trello.com/c/tRiFtYj5/984-reminders-response-driven-8)

**Review by NickGrantONS — APPROVED** [review]
Looks good 👍

File: Makefile [hunk 1]
```diff
@@ -2,7 +2,8 @@ install:
 	pipenv install --dev
 
 check:
-	PIPENV_PYUP_API_KEY="" pipenv check
+	# TODO: 38932 - Explicitly ignore (unused) cryptography package warning for now
```
Comment (AdamHawtin) [inline Makefile:5]: The cryptography package is used.

Comment (liamtoozer) [reply Makefile:5]: Sorry, I'll update my wording a bit better!

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**